src/libraries/System.Private.CoreLib/src/System/RuntimeType.cs

```diff
@@ -49,7 +49,7 @@ public override int GetArrayRank()
 
         public override object[] GetCustomAttributes(bool inherit)
         {
-            return CustomAttribute.GetCustomAttributes(this, ObjectType, inherit);
+            return CustomAttribute.GetCustomAttributes(this, (RuntimeType)typeof(object), inherit);
```

Comment:
This is technically a deoptimization, but the new intrinsic kicks in on so many places within custom attribute handling that I don't feel bad about deleting the static here.